sdk/communication/communication-phone-numbers/recordings/node/siproutingclient__set_domains/recording_can_set_an_existing_domain.json

```diff
@@ -0,0 +1,163 @@
+{
```

Comment:
This probably shouldn't be recorded? It's about setting domains and it ends with errorcode.